docs/source/glossary.rst

```diff
@@ -0,0 +1,161 @@
+Glossary
+========
+
+This glossary defines general Sawtooth terms and concepts.
+
+.. glossary::
+
+  Batch
+    Group of related transactions.
+    In Sawtooth, a batch is the atomic unit of state change for the blockchain.
+    A batch can contain one or more transactions. For a batch with multiple
+    transactions, if one transaction fails, all transactions in that batch fail.
+    (The client application is responsible for handling failure appropriately.)
+    For more information, see :doc:`architecture/transactions_and_batches`.
+
+  Blockchain
+    Distributed ledger that records transactions, in chronological order,
+    shared by all participants in a Sawtooth network. Each block on the
+    blockchain is linked by a cryptographic hash to the previous block.
+
+  Consensus
+    Process of building agreement among a group of mutually distrusting
+    participants (other nodes on a Sawtooth network). Sawtooth allows
+    different types of consensus on the same blockchain.
+    For more information, see :ref:`pluggable-consensus-label`.
+
+  Core
+    See :term:`Sawtooth core`.
+
+  Distributed ledger
+    See :term:`Blockchain`.
+
+  Genesis block
+    First block on the blockchain. The genesis block initializes the
+    Sawtooth network.
+
+  Global state
+    Database that stores a local (validator-specific) record of transactions for
+    the blockchain. Sawtooth represents state in a single instance of a
+    Merkle-Radix tree on each validator node.  For more information, see
+    :doc:`architecture/global_state`.
+
+  Identity
+    Sample transaction family that handles on-chain permissions (settings
+    stored on the blockchain) for transactor and validator keys. This
+    transaction family demonstrates how to streamline managing identities
+    for lists of public keys. For more information, see
+    :doc:`transaction_family_specifications/identity_transaction_family`.
+
+  IntegerKey
+    Sample transaction family with only three operations (set, increment, and
+    decrement) that can be used to test deployed ledgers. For more information,
+    see :doc:`transaction_family_specifications/integerkey_transaction_family`.
+
+  Journal
+    Sawtooth core component that is responsible for maintaining and extending
+    the blockchain for the validator.
```

Comment:
Should this link to the architecture/journal?

Reply:
Yes, it should. Thanks! I'll make that change.